server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -171,11 +171,7 @@ def to_representation(self, instance):
         """
         document_dict = {}
         for field in instance._fields:
-            if field in self._remapped_fields:
-                document_dict[self._remapped_fields[field]] = getattr(instance, field)
-            else:
-                document_dict[field] = getattr(instance, field)
-
+            document_dict[self.translate_field_reverse(field)] = getattr(instance, field)
```

Comment:
Looks like `translate_field_reverse()` passes `field` `to _remapped_fields()` without doing the `if field` as before.  Will  `to _remapped_fields()` do the right thing if `field=None`?